l5kit/l5kit/tests/sampling/deep_prediction_test.py

```diff
@@ -4,25 +4,11 @@
 import numpy as np
 import pytest
 
-from l5kit.configs import load_config_data
 from l5kit.data import AGENT_DTYPE, FRAME_DTYPE, ChunkedDataset
 from l5kit.rasterization import StubRasterizer
 from l5kit.sampling import generate_agent_sample
 
 
```

Comment:
These were never used in the previous version?

Reply:
yeah it was used to get a fixture here on line 20